source/guides/core-concepts/dashboard-service.md

```diff
@@ -228,7 +228,7 @@ All screenshots taken during the test run can be found in the **Screenshots** of
 
 The video recorded during the test run can be found under the **Video** of the spec. You can also download the video.
 
-{% url /img/dashboard/videos-of-recorded-test-run.png "Video of test runs" %}
+{% img /img/dashboard/videos-of-recorded-test-run.png "Video of test runs" %}
```

Comment:
Once the trailing quite was added the sections started being processed and this highlighted this wee issue with the wrong tag being used.

Simple to fix and then voila, all that sweet documentation started appearing 🎉 👍